src/App.jsx

```diff
@@ -46,11 +47,30 @@ const App = () => {
 
       console.log("Airtable API Response Data", data);
 
-      const todos = data.records.map((record) => ({
+      //ascending order
+      const sortedData =data.records.sort((a,b) => {
+        const titleA = a.fields.title ? a.fields.title.toLowerCase() : '';
+        const titleB = b.fields.title ? b.fields.title.toLowerCase() : '';
+
+        if (isAscending){
+          return titleA < titleB ? -1 : titleA >titleB ? 1:0;
+        }else{
+          return titleA<titleB ?1: titleA > titleB ? -1 :0;
+        }
+
+      });
```

Comment:
Cool approach! I've never seen ternaries used in a sort.

You want to be careful to keep consistent character spacing. Sometimes you're adding a space next to the colon and other times you are not. This makes it a little challenging to read. No need to fix it for this review but I wanted to raise your attention to it.